packages/react-components/react-utilities/src/trigger/types.ts

```diff
@@ -15,3 +17,14 @@
 export type FluentTriggerComponent = {
   isFluentTriggerComponent?: boolean;
 };
+
+/**
+ * A trigger may have a children that could be either:
+ * 1. A single element
+ * 2. A render function that will receive properties and must return a valid element or null
+ * 3. null or undefined
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type TriggerProps<TriggerChildProps = any> = {
```

Comment:
```suggestion
export type TriggerProps<TriggerChildProps = unknown> = {
```

Will `unknown` work there?